examples/input_output.jl

```diff
@@ -29,7 +29,7 @@ qe_input  = "Fe_afm.pwi"
 atoms     = load_atoms(qe_input)
 positions = load_positions(qe_input)
 lattice   = load_lattice(qe_input)
-magnetic_moments = load_magnetic_moments(qe_input)
+magnetic_moments = load_magnetic_moments(qe_input);
```

Comment:
I think it's nice to see these
```suggestion
magnetic_moments = load_magnetic_moments(qe_input)
```